app/services/api/presenters/event.rb

```diff
@@ -0,0 +1,56 @@
+module Api
+  module Presenters
+    class Event
+      attr_reader :event
+
+      def self.call(event)
+        new(event).call
+      end
+
+      def initialize(event)
+        @event = event
+      end
+
+      def call
+        {
+          id: event.id,
+          eventable_id: event.eventable_id,
+          eventable_type: event.eventable_type,
+          title: event.title,
+          event_type: event.event_type,
+          date: event.date,
+          description: event.description,
+          eventable_title: event.eventable_title,
+          link: link(event.eventable_type, event.eventable_id)
+        }
+      end
+
+      private
+
+      def link(eventable_type, eventable_id)
```

Comment:
following from the comment bellow we can pass to this method `event.eventable` and then just use that object. Then instead of `eql?` you can use `eventable.is_a?(Legislation)` etc.